tests/test_prompts_rewrite_all_skill.sh

```
@@ -0,0 +1,265 @@
+# ============================================
+
+assert_file_contains "ファイル先頭が --- で始まる frontmatter" "$SKILL_FILE" "^---$"
+assert_file_contains "name キーが prompts-rewrite-all" "$SKILL_FILE" "^name: prompts-rewrite-all$"
+assert_file_contains "description キーが存在" "$SKILL_FILE" "^description:"
+
+# ============================================
+echo "=== description にトリガー語句が 2 個以上含まれている ==="
+# ============================================
+
+# 「〜と言った時に使用」「〜と言われた時に使う」形式のトリガー語句
+TRIGGER_COUNT=$(grep -o '「[^」]\+」' "$SKILL_FILE" | head -1 | wc -l | tr -d ' ')
```

Shellcheck (0.11.0)

[warning] TRIGGER_COUNT appears unused. Verify use (or export if used externally).

(SC2034)